assets/react/components/header.js

```diff
@@ -18,8 +18,13 @@ module.exports = React.createClass({
 		}
 		return data;
 	},
-	myPosts: function(){
-		this.transitionTo('/my-posts');
+	myProfile: function(){
+		var user_id = this.state.user.id;
```

Comment:
How is this working? Isnt the state structure like:
{
  user: {
    users: [],
    currentUserId: 
}
}